app/assets/javascripts/app/views/comment_stream_view.js

```diff
@@ -52,11 +62,14 @@ app.views.CommentStream = app.views.Base.extend({
       success: function() {
         this.commentBox.val("");
         this.enableCommentBox();
+        this.mdEditor.hidePreview();
+        this.closeForm();
```

Comment:
There are already some tests for the behavior of this function, so please add tests for the added behavior.